thorlcr/master/thgraphmanager.cpp

```diff
@@ -711,6 +711,12 @@ void CJobManager::run()
         {
             factory.setown(getWorkUnitFactory());
             workunit.setown(factory->openWorkUnit(wuid));
+
+            unsigned maxLogDetail = workunit->getDebugValueInt("maxlogdetail", DefaultDetail); 
```

Comment:
Does lowercase matter here ?  Typically wouldn't it be maxLogDetail ?

Reply:
no, workunit debug values and the call, are case insensitive